sdk/logical/token.go

```diff
@@ -128,8 +134,13 @@ type TokenEntry struct {
 	CreationTimeDeprecated   int64         `json:"CreationTime" mapstructure:"CreationTime" structs:"CreationTime" sentinel:""`
 	ExplicitMaxTTLDeprecated time.Duration `json:"ExplicitMaxTTL" mapstructure:"ExplicitMaxTTL" structs:"ExplicitMaxTTL" sentinel:""`
 
+	// EntityID is the ID of the entity associated with this token.
 	EntityID string `json:"entity_id" mapstructure:"entity_id" structs:"entity_id"`
 
+	// If SkipIdentityInheritance is true, the token will not
```

Comment:
I have a mild preference for a name that contains "policies", e.g. `NoIdentityPolicies` or something like that.